Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsPage.cs

```diff
@@ -113,7 +113,7 @@ public NodejsIntellisenseOptionsPage()
             }
         }
 
-        internal bool OnlyTabOrEnterToCommit {
+        public bool OnlyTabOrEnterToCommit {
```

Comment:
no need to make this public - we should add InternalsVisibleTo if this is just for tests